examples/example-app-v11/src/environments/environment.ts

```diff
@@ -2,7 +2,9 @@
 // `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
 // The list of file replacements can be found in `angular.json`.
 
-export const environment = {
+import { AppEnvironment } from '../app/configs/environment.config';
```

Comment:
Only small thing is that the Angular example app doesn’t have this interface, we should remove this interface.

Reply:
@ahnpnl removed all sources that are not present in the tour of heroes